spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -382,7 +382,36 @@ public class MyApplication {
 
 ==== Query methods by convention
 
-This feature is not supported yet.
+[source, java]
+----
+public interface TradeRepository extends CrudRepository<Trade, String[]> {
+    List<Trader> findByAction(String action);
+
+  int countByAction(String action);
+
+  boolean existsByAction(String action);
+
+  List<Trade> findTop3ByActionAndSymbolAndPriceGreaterThanAndPriceLessThanOrEqualOrderBySymbolDesc(
+  			String action, String symbol, double priceFloor, double priceCeiling);
+}
+----
+
+In the example above, the {spring-data-commons-ref}/#repositories.query-methods[query methods]
+in `TradeRepository` are generated based on the name of the methods, using the
+{spring-data-commons-ref}#repositories.query-methods.query-creation[Spring Data Query creation
+naming convention].
+
+Cloud Datastore only supports filter components joined by AND, and the following operations:
+
+* `equals`
+* `greater than or equals`
+* `greater than`
+* `less than or equals`
+* `less than`
+* `is null`
+
+After writing a custom repository interface specifying just the signatures of these methods, you can
```

Comment:
"you can"??
Also, please stick to the one-sentence-per-line convention in `.adoc` files.

Reply:
@meltsufin changes made, PTAL